types/framework/entities/chatMessage.d.ts

```diff
@@ -169,7 +172,7 @@ declare class ChatMessage extends Entity<ChatMessage.Data> {
    *
    * @returns The identified speaker data
    */
-  static getSpeaker(speaker: DeepPartial<ChatMessage.SpeakerData>): ChatMessage.SpeakerData;
+  static getSpeaker(speaker?: ChatMessage.SpeakerCreateData): ChatMessage.SpeakerData;
```

Comment:
I also addressed this in https://github.com/League-of-Foundry-Developers/foundry-vtt-types/pull/256. As I suppose your change is more complete, it makes more seemed to go ahead with your PR. However, my PR also adds tests Frontals particular change. Could you maybe just copy them over to your PR?